internal/kafka/internal/migrations/migrations.go

```diff
@@ -90,6 +90,7 @@ var migrations = []*gormigrate.Migration{
 	renameKafkaBillingModelColumn(),
 	addExpiresAtToKafkaRequest(),
 	addClusterOrgIdClusterTypeColumns(),
+	renameKafkaStorageSizeColumn(),
```

Comment:
Let's make sure this is the last migration applied after applying rebases in this PR

Reply:
thanks Miguel, just updated the timestamp and the migrations.